src/falco/commands/start_project.py

```diff
@@ -97,6 +98,7 @@ def __call__(self) -> None:
             )
 
         self.init_project()
+        Htmx(version="latest", output=Path(self.project_name) / "static" / "js" / "htmx.min.js")()
```

Comment:
The `Htmx` command will exit the program if there is any network error, see https://github.com/Tobi-De/falco/blob/f32813a441c231d93d364ab55ab82b0a14c96d87/src/falco/commands/htmx.py#L30 and https://github.com/Tobi-De/falco/blob/f32813a441c231d93d364ab55ab82b0a14c96d87/src/falco/utils.py#L46, if that happens it will prevent the rest of the code from the `StartProject` code to run. I think we should [suppress](https://docs.python.org/3/library/contextlib.html#contextlib.suppress) any `cappa.Exit` raise by the command to avoid this :) .